metrics.go

```diff
@@ -45,6 +50,33 @@ func (c *ClientIMPL) metricsRequest(ctx context.Context, response interface{}, e
 	return err
 }
 
+// mirrorTransferRate - Volume Mirror Transfer Rate
+func (c *ClientIMPL) mirrorTransferRate(ctx context.Context, response interface{}, entityID string, limit int) error {
+	qp := getFSDefaultQueryParams(c)
+	qp.RawArg("id", fmt.Sprintf("eq.%s", entityID))
+	qp.Limit(limit)
+	qp.RawArg("select", "id,timestamp,synchronization_bandwidth,mirror_bandwidth,data_remaining")
+
+	customHeader := http.Header{}
+	customHeader.Add("DELL-VISIBILITY", "Internal")
+	apiClient := c.APIClient()
+	apiClient.SetCustomHTTPHeaders(customHeader)
```

Comment:
@nitesh3108 Shouldn't we unset this custom header at the last?

Reply:
Yes, its done